PayPalWebPayments/src/main/java/com/paypal/android/paypalwebpayments/PayPalWebCheckoutClient.kt

```diff
@@ -78,69 +73,67 @@ class PayPalWebCheckoutClient internal constructor(
      * @param request [PayPalWebCheckoutRequest] for requesting an order approval
      */
     fun start(request: PayPalWebCheckoutRequest) {
-        analyticsService.sendAnalyticsEvent("paypal-web-payments:started", orderId)
-
-        CoroutineScope(dispatcher).launch(exceptionHandler) {
-            try {
-                val browserSwitchOptions = browserSwitchHelper.configurePayPalBrowserSwitchOptions(
-                    request.orderId,
-                    coreConfig,
-                    request.fundingSource
-                )
-                browserSwitchClient.start(activity, browserSwitchOptions)
-            } catch (e: PayPalSDKError) {
-                deliverFailure(APIClientError.clientIDNotFoundError(e.code, e.correlationId))
-            }
+        analyticsService.sendAnalyticsEvent("paypal-web-payments:started", request.orderId)
+        payPalWebLauncher.launchPayPalWebCheckout(activity, request)?.let { launchError ->
+            notifyWebCheckoutFailure(launchError)
         }
     }
 
+    fun vault(activity: AppCompatActivity, request: PayPalWebVaultRequest) {
```

Comment:
Still working on docs, but I wanted to get the fundamental structure going before adding that last piece of polish.